pkgs/by-name/sd/sdl3/package.nix

```
@@ -0,0 +1,221 @@
+{
+  lib,
+  stdenv,
+  config,
+  alsa-lib,
+  apple-sdk_11,
+  cmake,
+  darwinMinVersionHook,
+  dbus,
+  fcitx5,
+  fetchFromGitHub,
+  ibus,
+  installShellFiles,
+  libGL,
+  libdecor,
+  libdrm,
+  libjack2,
+  libpulseaudio,
+  libusb1,
+  libxkbcommon,
+  mesa,
+  nas,
+  ninja,
+  nix-update-script,
+  nixosTests,
+  pipewire,
+  sndio,
+  systemdLibs,
+  testers,
+  validatePkgConfig,
+  wayland,
+  wayland-scanner,
+  xorg,
+  zenity,
+  alsaSupport ? stdenv.hostPlatform.isLinux && !stdenv.hostPlatform.isAndroid,
+  dbusSupport ? stdenv.hostPlatform.isLinux && !stdenv.hostPlatform.isAndroid,
+  drmSupport ? stdenv.hostPlatform.isLinux && !stdenv.hostPlatform.isAndroid,
+  ibusSupport ? stdenv.hostPlatform.isUnix && !stdenv.hostPlatform.isDarwin,
+  jackSupport ? stdenv.hostPlatform.isLinux && !stdenv.hostPlatform.isAndroid,
+  libdecorSupport ? stdenv.hostPlatform.isLinux && !stdenv.hostPlatform.isAndroid,
+  openglSupport ? lib.meta.availableOn stdenv.hostPlatform mesa,
+  pipewireSupport ? stdenv.hostPlatform.isLinux && !stdenv.hostPlatform.isAndroid,
+  pulseaudioSupport ?
+    config.pulseaudio or stdenv.hostPlatform.isLinux && !stdenv.hostPlatform.isAndroid,
+  libudevSupport ? stdenv.hostPlatform.isLinux && !stdenv.hostPlatform.isAndroid,
+  sndioSupport ? false,
+  testSupport ? true,
+  waylandSupport ? stdenv.isLinux && !stdenv.hostPlatform.isAndroid,
+  x11Support ? !stdenv.hostPlatform.isAndroid && !stdenv.hostPlatform.isWindows,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "sdl3";
+  version = "3.1.6";
+
+  outputs = [
+    "lib"
+    "dev"
+    "out"
+  ];
+
+  src = fetchFromGitHub {
+    owner = "libsdl-org";
+    repo = "SDL";
+    rev = "refs/tags/preview-${finalAttrs.version}";
```

```suggestion
    tag = "preview-${finalAttrs.version}";
```
You can use this new `tag` parameter (not a blocker).